registrations/tests/test_price_group_get.py

```diff
@@ -243,31 +245,42 @@ def test_price_group_list_ordering(api_client):
     api_client.force_authenticate(user)
 
     price_groups = [
-        PriceGroupFactory(description=""),
-        PriceGroupFactory(description="1"),
-        PriceGroupFactory(description="2"),
-        PriceGroupFactory(description="Abc"),
-        PriceGroupFactory(description="Bcd"),
-        PriceGroupFactory(description="Cde"),
-        PriceGroupFactory(description="Äää"),
-        PriceGroupFactory(description="Öää"),
-        PriceGroupFactory(description="Ööö"),
+        PriceGroupFactory(description_fi=""),
+        PriceGroupFactory(description_fi="1"),
+        PriceGroupFactory(description_fi="2"),
+        PriceGroupFactory(description_fi="Abc"),
+        PriceGroupFactory(description_fi="Bcd"),
+        PriceGroupFactory(description_fi="Cde"),
+        PriceGroupFactory(description_fi="Dce"),
+        PriceGroupFactory(description_fi="Äää"),
+        PriceGroupFactory(description_fi="Äöö"),
+        PriceGroupFactory(description_fi="Öää"),
+        PriceGroupFactory(description_fi="Ööö"),
     ]
 
-    page_size = 3
-    page_start = 0
-    page_end = page_size
-    for page in range(1, 4):
-        response = get_list(api_client, query=f"page={page}&page_size={page_size}")
+    # Price groups are ordered by description, so need to activate the correct
+    # language.
+    activate("fi")
```

Comment:
Using `django.utils.translation.override()` would be better to make sure other tests aren't affected.